diff --git a/unused-dependencies/build.gradle b/unused-dependencies/build.gradle
--- a/unused-dependencies/build.gradle
+++ b/unused-dependencies/build.gradle
@@ -2,7 +2,5 @@
 description = "Gradle Unused Dependencies example"
 
 dependencies {
-    implementation('com.google.guava:guava:29.0-jre')
-    implementation('org.springframework.boot:spring-boot:2.3.1.RELEASE')
     testImplementation('junit:junit:4.12')
 }
